File: cases/taint/php/infeasible-branch-positive/infeasible_branch_positive.php
<?php
function dfb_source(): string { // DFB-SOURCE: infeasible-branch-input
    return "tainted";
}

function dfb_sink(string $value): void {} // DFB-SINK: infeasible-branch-sink

function run(): void {
    $value = "clean";
    if (true) {
        $value = dfb_source(); // DFB-WITNESS: feasible-tainted-branch
    }
    dfb_sink($value);
}

File: cases/taint/rust/infeasible-branch-positive/infeasible_branch_positive.rs
fn dfb_source() -> i32 { // DFB-SOURCE: infeasible-branch-input
    1
}

fn dfb_sink(value: i32) {} // DFB-SINK: infeasible-branch-sink

fn run() {
    let mut value = 0;
    if true {
        value = dfb_source(); // DFB-WITNESS: feasible-tainted-branch
    }
    dfb_sink(value);
}
